samples/matter/lock/src/credentials/credentials_manager.h

```diff
@@ -134,6 +134,19 @@ template <DoorLockData::CredentialsBits CRED_BIT_MASK> class CredentialsManager
 			   DlCredentialStatus credentialStatus, CredentialTypeEnum credentialType,
 			   const chip::ByteSpan &secret);
 
+	DlStatus GetWeekDaySchedule(uint8_t weekdayIndex, uint16_t userIndex,
```

Comment:
Unfortunately, if we want to be consistent, all of these new methods should have doxygen commentary :)

Reply:
Done